tests/unit/test_index.py

```diff
@@ -84,6 +84,9 @@ def test_ext_fragment(self):
     def test_ext_query(self):
         assert '.whl' == Link('http://yo/wheel.whl?a=b').ext
 
+    def test_is_wheel(self):
+        assert Link('http://yo/wheel.whl').is_wheel
```

Comment:
well checking for `True` would allow the test to crash if someone removes the `property` decorator